source/_integrations/tesla_fleet.markdown

```
@@ -358,6 +358,67 @@
+| supercharger\_id | Tesla Supercharger site ID (found via API sniffing, see notes below) | 1389                             |
+| order            | (Optional) Order of this waypoint                                    | 0                                |
+
+Please be aware this will initiate precondtitioning if required. There is no "cancel" command if you decide to reverse this action, but you could send a navigation request (`navigation_request`) to your home address which should clear it.
+
+#### Finding the Supercharger ID
+
+The `supercharger_id` used for the `navigate_to_supercharger_request` is not publicly documented by Tesla. To find it:
+
+1. Use **[mitmproxy](https://mitmproxy.org/)** to monitor your Tesla mobile app.
+2. Open the Tesla app and navigate to **Locations > Charging**.
+4. Select a Supercharger and observe the network requests.
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 3; Actual: 4; Style: 1/2/3

(MD029, ol-prefix)